### DISCUSSION TITLE
Gitea Themes : possible to make them work with Gitea-Modern theme ?

### DISCUSSION THREAD

**Discussion by MilesTEG1** [themepark-dev/theme.park#405] [Q&A]
Hello @GilbN ,
I love your themes, especially for Gitea and for Vaultwarden :) So thanks for your work.

I just found on gitea's documentation, a new theme : Gitea-Modern ( [https://codeberg.org/Freeplay/Gitea-Modern](https://codeberg.org/Freeplay/Gitea-Modern) ) witch give a new look (not only colors) for Gitea :
![](https://codeberg.org/Freeplay/Gitea-Modern/media/branch/main/images/codeberg-modern_repoPg.png)

And this new display is great, and I'm wondering if it can be possible to make it work with your themes ?
For now, if I setup this modern theme into Stylus, with your Dracula theme (installed as your documentation says), I get this :
![image](https://user-images.githubusercontent.com/29298312/182035139-0ad08d91-4e5b-4b45-a9eb-978f31aa6ed4.png)

Without the stylus modern-theme set, I get :
![image](https://user-images.githubusercontent.com/29298312/182035164-821460fd-7eb2-4add-b685-c9d6d4886b54.png)

It would be so great to have the two themes working together :)

Thanks for your help.

**Comment by GilbN** [comment]
Hi. Looks like a nice theme. 
It's probably possible, but it's not something I'm going to spend time on as I don't use Gitea. 

But if you want to make a gitea-modern theme and make a PR, I'm willing to add it.
